README.md

```diff
@@ -55,34 +67,27 @@ cd notebooks
 ```
 
 ## Run Planet Notebooks in Docker
-Planet Notebooks relies on a complex stack of technologies that are not always easy to install and properly 
+Planet Notebooks rely on a complex stack of technologies that are not always easy to install and properly 
 configure. To ease this complexity we provide a docker container for running the notebook on docker compatible 
 systems. To install docker on your system please see docker's [documentation](https://docs.docker.com/engine/installation/)
 for your operating system.
 
 ### Download prebuilt Docker image (recommended)
-The Docker image for these notebooks is hosted in the [planetlabs/notebooks](https://hub.docker.com/r/planetlabs/notebooks)
-repo on DockerHub. To download and prepare the image for use, run:
+The Docker image for these notebooks is hosted in the [planetlabs/notebooks](https://hub.docker.com/r/planetlabs/notebooks) repo on DockerHub. To download and prepare the image for use, run:
+
 ```bash
+cd notebooks
```

Comment:
not all users have their repo folder the same as the repo name (i.e. 'notebooks'). Consider generalizing this, e.g. with `<notebook repo directory>'?

Reply:
In the docs above the user is instructed to checkout the notebooks into a folder named `notebooks`:

```
git clone https://github.com/planetlabs/notebooks.git
cd notebooks
```

I just added this line above this section, because I had left the folder and was surprised when it failed for me. I thought it ws a good reminder for other users.